### PR TITLE
Feature/macgills/1275 duplicate strings multi module

### LINKED ISSUE #1275: Lint Issue: Duplicate Strings
**Is your feature request related to a problem? Please describe.**
```
Duplicate Strings
 Duplicate strings can make applications larger unnecessarily.
 This lint check looks for duplicate strings, including differences for strings where the only difference is in capitalization. 
Title casing and all uppercase can all be adjusted in the layout or in code.  
Issue id: DuplicateStrings
```

**Describe the solution you'd like**
This should be fixed to reduce app size and lessen the burden of translation. It will require a regeneration of translations after the issue is addressed for English.

### PR DESCRIPTION
**Pull request by macgills**
Fixes #1275

### CODE REVIEW

**Comment by macgills** [comment]
@Abijeet this PR should be much friendlier for TW to deal with as I have not moved any translations

There are still 13 duplicates, viewable in this [report](https://drive.google.com/open?id=1yxLWPYFxe1MdTwD6JKBLdmwMr5Mpfv6Q), that I am unsure of how to deal with.

I can ignore them for now but I think a few of these cases are in error. Is there a process for me to highlight that on TW's side? Hopefully we will update all the missing cases in `qq` soon, it may even be possible to configure a lint check such that it will be impossible for us to not specify it

**Comment by Abijeet** [comment]
There are two options here,

1. Delete the translations, not many duplicates are there, so it's not a big deal.
2. Add a `fuzzy="true"` attribute to the translation element. eg: `<string name="menu_zim_manager" fuzzy="true">Obter contido</string>`. This will highlight to translators that the translation is outdated and needs to be updated. Translators will then validate the translations, and see if the duplicates are valid translations or not. Until then though you'll have to ignore them.

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
           fetch-depth: 1
 
       - name: Static Analysis
-        run: ./gradlew ktlintCheck app:lintDebug core:lintDebug custom:lintCustomexampleDebug
+        run: ./gradlew ktlintCheck app:lintDebug custom:lintCustomexampleDebug
```
Comment (abdulwd) [inline .github/workflows/pull_request.yml]: Why has this changed?

Comment (macgills) [reply .github/workflows/pull_request.yml]: We are now linting core as part of app by using `lintOptions.isCheckDependencies=true`

This gives us a more accurate lint report. Core contains strings only used in App so when you lint core by itself it says "unused", the ideal solution would be to migrate the strings+translations to their appropriate module but that is not possible at the moment, I've created a ticket #1788 to get the end result we want

**Pull request merged**